geeksay.js

```diff
@@ -83,9 +83,23 @@ const translations = {
     "everything": "42",
     "lifetime": "runtime",
     "sleep": "shutdown",
+    "black": "#000000",
+    "silver": "#c0c0c0",
+    "gray": "#808080",
+    "grey": "#808080",
+    "white": "#ffffff",
+    "maroon": "#800000",
     "red": "#ff0000",
-    "green": "#00ff00",
+    "purple": "#800080",
+    "fuchsia": "#ff00ff",
```

Comment:
The translations are meant to be easily understood. Do you think you can read #ff00ff and decode it as fuchsia?

Reply:
Ah that's true! I'll take it out.